perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -171,6 +178,27 @@
 LOGIN_REDIRECT_URL = '/manage/create/'
 LOGIN_URL = '/login'
 
+
+# Axes, fundamental integration
+AUTHENTICATION_BACKENDS = [
+    # AxesBackend should be the first backend in the AUTHENTICATION_BACKENDS list.
+    'axes.backends.AxesBackend',
+
+    # Django ModelBackend is the default authentication backend.
+    'django.contrib.auth.backends.ModelBackend',
+]
+AXES_LOCKOUT_TEMPLATE = 'registration/login_attempts_exceeded.html'
+
+# Axes, configure behavior
+AXES_ENABLED = True
+AXES_FAILURE_LIMIT = 6
+AXES_LOCK_OUT_AT_FAILURE = True
+AXES_COOLOFF_MINUTES = 30
+AXES_COOLOFF_TIME = 'perma.utils.cooloff_time'
+AXES_ONLY_USER_FAILURES = True  # If True, only lock based on username, and never lock based on IP if attempts exceed the limit. Otherwise utilize the existing IP and user locking logic. Default: False
+AXES_RESET_ON_SUCCESS: True  # If True, a successful login will reset the number of failed logins. Default: False
```

Comment:
`:` should be `=`

Reply:
Oh wow! I'm weirded out that nothing seems broken with it this way.

Reply:
Whoa, it's a type annotation!

```
>>> AXES_RESET_ON_SUCCESS: True
>>> __annotations__
{'AXES_RESET_ON_SUCCESS': True}
```

See also https://www.python.org/dev/peps/pep-0526/#other-uses-of-annotations where they're like "but don't do that." :)

Reply:
🙌